docs/howto/object-storage/s3/credentials.md

```diff
@@ -45,20 +45,32 @@ need to configure your S3 client with it.
 How exactly you do that depends on your preferred client:
 
 === "aws"
-    Create a new profile, named after your {{brand}} region:
+    Install the [`endpoint`](https://pypi.org/project/awscli-plugin-endpoint/) plugin:
+    ```bash
+    pip install awscli-plugin-endpoint
+    ```
+    Then, enable the plugin in your configuration.
+    Open the `aws` configuration file (normally `~/.aws/config`), and add or modify the `[plugins]` section:
+    ```ini
+    [plugins]
+    endpoint = awscli_plugin_endpoint
+    ```
+    Now, create a new profile, named after your {{brand}} region.
```

Comment:
```suggestion
    Now, create a new profile named after your {{brand}} region.
```

Reply:
I like it better with the additional comma and would like to keep this as-is. :)